handlers/health.go

```diff
@@ -30,7 +30,7 @@ func HealthCheckHandlerCreator() func(http.ResponseWriter, *http.Request) {
 		res, err := elasticsearch.GetStatus()
 		if err != nil {
 			healthIssue = err.Error()
-		} else if !strings.Contains(string(res), " green ") {
+		} else if !(strings.Contains(string(res), " green ") || strings.Contains(string(res), " yellow ")) {
```

Comment:
🤔Personally I think this looks a bit nasty. I would consider pulling this out into its own function with a nice readable name to make this `if-else` read like English.